src/error_message.ts

```diff
@@ -1,6 +1,7 @@
 import { FolderID } from './types';
 
 export const errorMessage = {
+	cannotUseDuplicateNameInParentFolder: 'Parent folder already has an entity with the name!',
```

Comment:
How about a constant name like `entityNameExists` and a message like `Entity name already exists in destination folder!"